### PR TITLE
Improved German translation

### PR DESCRIPTION
**Pull request by Atalanttore**
I've replaced some anglicisms and made the German translation a little bit more consistent.

### CODE REVIEW

File: strings.txt [hunk 1]
```diff
@@ -8925,7 +8927,7 @@
     hu = %@ MB-t kell letöltened, de csak %@ MB hely szabad.
     el = Πρέπει να κάνετε λήψη %@ MB, αλλά μόνο %@ MB είναι διαθέσιμα.
     ar = تحتاج إلى تحميل %@ ميجابايت، ولكن لا يتوافر سوى %@ ميجا بايت.
-    de = Der Download erfordert %@ MB, es sind jedoch nur %@ MB verfügbar.
+    de = Sie müssen %@ MB herunterladen, aber es sind nur %@ MB verfügbar.
```
Comment (hybridOL) [inline strings.txt]: This one should be improved. The new one sounds as if the download does not provide enough MB. Maybe better:
Sie wollen %@ MB herunterladen, der Speicherplatz reicht aber nur für %@ MB.

**Comment by deathbaba** [comment]
Thanks!
Please check and sign our CLA http://maps.me/cla/ before merge.
And this instruction shows how to correctly update app interface strings after modifying strings.txt https://github.com/mapsme/omim/wiki/strings.txt

File: strings.txt (hunk 1 above)
Comment (Atalanttore) [inline strings.txt]: According to your suggestion, I have improved this string.

**Comment by Atalanttore** [comment]
CLA is checked and signed.

**Comment by deathbaba** [comment]
Great! Thank you very much!

**Comment by trashkalmar** [comment]
@deathbaba What about generated strings for platforms?

**Comment by deathbaba** [comment]
Oops, forgot to recheck it. #2915 contains generated strings.

**Pull request merged**